common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,91 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts_with_max(self):
```

Comment:
All tests should have comments and the first line of the comment should be meaningful.

The first line of the comment is printed when the test fails so that line should ideally be a compete sentence or idea.  It should differentiate the test from other tests like it but also be similar.  You can use subsequent lines to further explain the test if needed.  You have 50 or 80 characters on the first line so use them. 

Below are possible comment examples for the first three tests.  When I make a group of tests I will try to be consistent with the comments and function names.  Maybe you'll have a better idea for having similar comments since you're more familiar with the test group.

Show answer button should NOT be available when attempts is less than attempts before.
Show answer button should NOT be available when attempts is less than attempts before.
Show answer button should be available when attempts > attempts_before.